03.bowling/bowl.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = []
+shots.each_slice(2) do |s|
+  frames << if s[0] == 10
+              [s[0]]
+            else
+              s
+            end
+end
+
+point = 0
+
+(0..frames.size - 1).each do |i|
+  point += if i < 9 # 9フレームまでの処理
+             if (frames[i][0] == 10) && !frames[i + 1].nil? # ストライク
```

Comment:
```suggestion
             if frames[i][0] == 10
```

`&& !frames[i + 1].nil?` って必要ですか？